packages/explorer/src/examples/react-example/package.json

```diff
@@ -9,8 +9,8 @@
   "dependencies": {
   },
   "alias": {
-    "react": "../../../../node_modules/react",
-    "react-dom": "../../../../node_modules/react-dom",
+    "react": "../../../../../node_modules/react",
+    "react-dom": "../../../../../node_modules/react-dom",
     "scheduler/tracing": "../../../node_modules/scheduler/tracing-profiling"
```

Comment:
This scheduler should be updated to "../../../../../node_modules/scheduler/tracing-profiling" as well, right?

Reply:
this can actually be deleted I think - we don't use it